README.md

```diff
@@ -0,0 +1,38 @@
+# deno_postgres_interop
+[![style: solid](https://img.shields.io/badge/style-solid-orange)](https://pub.dev/packages/solid_lints)
+[![$solid_lints](https://nokycucwgzweensacwfy.supabase.co/functions/v1/get_project_badge?projectName=deno_postgres_interop)](https://www.worklog.ai)
+
+This package was created to be used in supabase edge functions
+because existing db-driver/orm packages depend on dart:io.
+It should also work for other cases with dart2js.
```

Comment:
Package allows to use [Deno Postgres](https://deno.land/x/postgres) in [Deno Deploy ](https://deno.com/deploy) on [dart_edge](https://docs.dartedge.dev/) by creating interop bindings for js library, since other drivers rely on dart:io and doesn't work there.

Main scenario Supabase Edge Functions, but it should also work for other cases with dart2js.